screwdriver/screwdriver.go

```diff
@@ -52,7 +54,7 @@ type API interface {
 	UpdateStepStop(buildID int, stepName string, exitCode int) error
 	SecretsForBuild(build Build) (Secrets, error)
 	GetAPIURL() (string, error)
-	GetCoverageInfo(jobID, pipelineID int, jobName, pipelineName string) (Coverage, error)
+	GetCoverageInfo(jobID, pipelineID int, jobName, pipelineName, scope, prNum, prParentJobId string) (Coverage, error)
```

Comment:
is `prParentJobId` int or string?

Reply:
it’s string.  I’m type casting before passing it. Go is statically  typed .